### PR TITLE
Add 2020 backtax option

### PR DESCRIPTION
**Pull request by embarnard**

### CODE REVIEW

File: app/models/intake.rb [hunk 1]
```diff
@@ -453,6 +453,7 @@ def filing_years
   end
 
   def most_recent_filing_year
+    # TODO: should the default be changed to 2020?
```
Comment (embarnard) [inline app/models/intake.rb:456]: Question for reviewer

Comment (bytheway875) [reply app/models/intake.rb:456]: yes! I think this should be updated to 2020!

File: app/views/questions/life_situations/edit.html.erb [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<% content_for :page_title, t('views.questions.life_situations.title') %>
+<% content_for :page_title, t('views.questions.life_situations.title', year: current_intake.most_recent_filing_year ) %>
```
Comment (bytheway875) [inline app/views/questions/life_situations/edit.html.erb:1]: Awesome! I'm glad you were able to incorporate this change.

File: spec/services/mixpanel_service_spec.rb [hunk 3]
```diff
@@ -280,14 +283,14 @@
 
           it "includes stimulus triage data" do
             expect(data_from_intake).to include(
```
Comment (bytheway875) [inline spec/services/mixpanel_service_spec.rb:285]: yay! I enjoy the re-indentation.

**Review by bytheway875 — APPROVED** [review]
Looks good to me! The only outstanding question I have is -- are we continuing to do 2017 this year? Or do we need to remove 2017 once we add 2020 because VITA only preps x years each year?

**Comment by embarnard** [comment]
> Looks good to me! The only outstanding question I have is -- are we continuing to do 2017 this year? Or do we need to remove 2017 once we add 2020 because VITA only preps x years each year?

@bytheway875 That's a good question! I checked with Yvonne and it looks like 2017 should be an option until April 15th. Thanks for the review!

**Pull request merged**